h2o-py/h2o/automl/autoh2o.py

```diff
@@ -344,8 +344,19 @@ def _fetch(self):
         else:
             self._leader_id = None
 
-        # Parse leaderboard H2OTwoDimTable & return as an H2OFrame
-        leaderboard = h2o.H2OFrame(res["leaderboard_table"].cell_values, column_names=res["leaderboard_table"].col_header)
+        # Intentionally mask the progress bar here since showing multiple progress bars is confusing to users.
+        # If any failure happens, revert back to user's original setting for progress and display the error message.
+        try:
```

Comment:
Where is the progress bar turned off?

Reply:
Oops forgot to push the other commit. Thanks!